### PR TITLE
Possibleerrorfix

### PR DESCRIPTION
**Pull request by phoenixking25**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+# }
 HOST = '0.0.0.0'
 PORT = 8080
-sqlite = {
-    'CREATE_ENGINE_URL': 'sqlite:///{}'.format(DB_PATH)
-}
\ No newline at end of file
+SQLALCHEMY_DATABASE_URI = 'mysql+pymysql://root@localhost/obscura' #username:password@host/database_name
+SQLALCHEMY_TRACK_MODIFICATIONS = False
+PASS_SECRET = 'phoenix'
```
Comment (AnshulMalik) [inline config.py]: I don't know where are these three constants being used?

File: config.py [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+mysql = {
+    'CREATE_ENGINE_URL': 'mysql+pymysql://root@localhost/obscura' #username:password@host/database_name
```
Comment (AnshulMalik) [inline config.py]: It should take this URL from environment and fallback to default if not present

File: controller.py [hunk 3]
```diff
@@ -13,8 +13,8 @@ def connect_to_database():
         Connect to our SQLite database and return a Session object
         """
 
-        engine = create_engine(config.sqlite['CREATE_ENGINE_URL'], echo=True)
+        engine = create_engine(config.mysql['CREATE_ENGINE_URL'], echo=True)
         engine.raw_connection().connection.text_factory = lambda x: x.encode('utf-8', 'ignore')
         session = scoped_session(sessionmaker(bind=engine))
         DeclarativeBase.query = session.query_property()
-        return session()
+        return session()
```
Comment (AnshulMalik) [inline controller.py:20]: EOL

**Review by AnshulMalik — CHANGES_REQUESTED** [review]

File: config.py (hunk 1 above)
Comment (phoenixking25) [reply config.py]: No where :P

File: config.py (hunk 2 above)
Comment (phoenixking25) [reply config.py]: ok

File: controller.py (hunk 3 above)
Comment (phoenixking25) [reply controller.py:20]: Whats is EOL

**Review by AnshulMalik — APPROVED** [review]

**Pull request merged**